guide/items.md

```diff
@@ -1,5 +1,16 @@
 ## Items
 
+`extern crate` statements must be first in a file. They must be ordered
+alphabetically.
+
+`use` statements, and module *declarations* (`mod foo;`, not `mod { ... }`)
+must come before other items. We recommend that imports come before module
+declarations; if imports and modules are separated, then they should be ordered
```

Comment:
What happens to relative imports like `super::` and `self::` should they be sorted under S? Also many people prefer the three sections rule. Maybe allow this by stating that there may be different sections of imports each separated by a blank line with the lines in a section sorted alphabetically.